ci/Jenkinsfile.android

```diff
@@ -2,7 +2,7 @@ pipeline {
   agent { 
     docker {
       label 'linux'
-      image 'statusteam/status-build-android:1.1.0'
+      image 'statusteam/status-build-android:1.1.0-16a42e06' /* remember to keep this up-to-date with the value in docker/android/Makefile */
```

Comment:
You canmove this comment to before the line instead of in the line, easer to read.  Maybe make it a `WARNING:`?